UC-Berkeley-Extension/global
# Info card and row components

## Asana Ticket(s)
- https://app.asana.com/0/0/1196473750676367/f
- https://app.asana.com/0/0/1196473750676370/f

## Description
- Styleguide creation of Info Card component and Info Row component
- Drupal paragraphs creation of Info Card component and Info Row component
- Drupal creation of image style to be used for Info Card
- Theming appropriate paragraph template to match the styleguide

## Testing instructions
- Serve the styleguide (navigate to the styleguide dir, run `npm start`)
- Check molecules -> Info Card and organisms -> Info Row, confirm the styling
- TODO: mobile styling

- `ddev start` on project root
- `ddev ssh` into the container
- `cd ..` then run `scripts/refresh.sh`. This should import config & clear cache.
- Create a landing page (https://global.ddev.site/node/add/landing_page), add Info Row component.
- Make sure the component renders properly and matches the styleguide.

## Remaining tasks
- [ ] mobile styling
